Formula/graph-tool.rb

```diff
@@ -84,13 +84,20 @@ class GraphTool < Formula
     sha256 "52de08355fd5cfb3ef4533891092bb96229d43c2069703d4aff04fdbedf9c92f"
   end
 
+  def python3
+    deps.map(&:to_formula)
+        .find { |f| f.name.match?(/^python@\d\.\d+$/) }
+        .opt_bin/"python3"
+  end
+
   def install
     # Linux build is not thread-safe.
     ENV.deparallelize unless OS.mac?
 
-    system "autoreconf", "-fiv"
-    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
-    venv = virtualenv_create(libexec, Formula["python@3.9"].opt_bin/"python3")
+    system "autoreconf", "--force", "--install", "--verbose"
+    site_packages = Language::Python.site_packages(python3)
+    xy = Language::Python.major_minor_version(python3)
+    venv = virtualenv_create(libexec, python3)
 
     resources.each do |r|
       venv.pip_install_and_link r
```

Comment:
This doesn't look right.

Reply:
Probably can clean up. Right now I don't think any `graph-tool` PyPI resource installs into `libexec/bin` so this isn't causing unwanted symlinks (i.e. `_and_link` is no-op).